website/docs/docs/cloud/about-cloud-develop-defer.md

```diff
@@ -37,10 +37,13 @@ When using defer, it compares artifacts from the most recent successful producti
 
 To enable defer in the dbt Cloud IDE, toggle the **Defer to production** button on the command bar. Once enabled, dbt Cloud will:
 
-1. Pull down the most recent manifest from the Production environment for comparison
-2. Pass the `--defer` flag to the command (for any command that accepts the flag)
+1. Configure deploy jobs across all three staging environments.
```

Comment:
it might be me but are we saying users need to configure 3 staging environments?